### PR TITLE
Update clamav version to 0.104.2

### PR DESCRIPTION
**Pull request by steverendell**

### CODE REVIEW

File: .github/workflows/clamav-daily.yml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 name: Twice daily build ClamAV with up-to-date virus DB
 on:
-  push:
```
Comment (steverendell) [inline .github/workflows/clamav-daily.yml:3]: not needed as part of build

File: .github/workflows/docker-build-push.yml [hunk 2]
```diff
@@ -51,19 +51,9 @@ jobs:
...
-      - name: Run Trivy vulnerability scanner
-        uses: aquasecurity/trivy-action@master
-        with:
-          image-ref: ${{ env.REGISTRY }}/${{ github.repository_owner }}/${{ matrix.dockerfile-dir }}:${{ github.sha }}
-          format: 'table'
-          exit-code: '1'
-          ignore-unfixed: true
-          vuln-type: 'os,library'
-          severity: 'CRITICAL,HIGH'
-
```
Comment (steverendell) [inline .github/workflows/docker-build-push.yml:63]: not needed as part of build - scan is done regularly anyway

File: hmpps-clamav/Dockerfile [hunk 3]
```diff
@@ -1,74 +1,12 @@
-FROM centos:latest
+FROM clamav/clamav:stable_base
```
Comment (steverendell) [inline hmpps-clamav/Dockerfile:1]: this is the clamav-supplied latest version

File: hmpps-clamav/docker-entrypoint.sh [hunk 4]
```diff
@@ -24,8 +24,8 @@ function process_file() {
     fi
 }
 
-process_file "${CLAMD_SETTINGS_CSV}" /usr/local/etc/clamd.conf
-process_file "${FRESHCLAM_SETTINGS_CSV}" /usr/local/etc/freshclam.conf
+process_file "${CLAMD_SETTINGS_CSV}" /etc/clamav/clamd.conf
+process_file "${FRESHCLAM_SETTINGS_CSV}" /etc/clamav/freshclam.conf
```
Comment (steverendell) [inline hmpps-clamav/docker-entrypoint.sh:28]: match paths in clamavs image

**Comment by swilson-mojhub** [comment]
Looks good, using the `clamav` has made the dicker image much easier to read.

**Review by swilson-mojhub — APPROVED** [review]
Approved

**Pull request merged**